app/src/main/java/org/mozilla/focus/biometrics/BiometricAuthenticationDialogFragment.kt

```diff
@@ -93,6 +95,8 @@ class BiometricAuthenticationDialogFragment : AppCompatDialogFragment(), Lifecyc
 
     private fun biometricNewSessionButtonClicked() {
         requireComponents.tabsUseCases.removePrivateTabs()
+        requireComponents.appStore.dispatch(AppAction.ShowHomeScreen)
+        topSitesIntegration.deleteAllTopSites()
```

Comment:
@iorgamgabriel When the 'New session' button is clicked, we remove the users top sites? Those are persistent through a session. Is that really what we want?

You posted this in the linked issue, but it's not clear to me if @jeffreygee acknowledged it.
> When the users clicks on "New Session" button from Lockscreen all the shortcuts will be deleted.

Raising this because, as a user, I would not want and have expected this to happen to my shortcuts.

Reply:
If it is intentional, maybe we need to add some copy that tells the user that turning on fingerprint to unlock can cause you to lose your shorcuts?

Reply:
@jonalmeida If the lockscreen appears if you have one shortcut I assume that this is a security issue . So if the user can't login in the app the shortcut will be deleted. I documented in the task . https://github.com/mozilla-mobile/focus-android/issues/6166#issuecomment-1092780991